readthedocs/search/faceted_search.py

```diff
@@ -71,10 +114,9 @@ def query(self, search, query):
 
         # need to search for both 'and' and 'or' operations
         # the score of and should be higher as it satisfies both or and and
-
         for operator in self.operators:
-            query_string = SimpleQueryString(
-                query=query, fields=self.fields, default_operator=operator
+            query_string = self._get_text_query(
+                query, self.fields, operator,
```

Comment:
I like passing kwargs to all functions, it makes things more explicit in my experience.